src/main/java/net/helpscout/api/ApiClient.java

```diff
@@ -563,7 +594,7 @@ public String getThreadSource(Long conversationID, Long threadID) throws ApiExce
      * @throws ApiException
      * @deprecated use {@link #getAttachmentData(Long)}
      */
-    public String getAttachmentData(Integer attachmentID) throws ApiException {
+    public String getAttachmentData(Integer attachmentID) throws IOException {
```

Comment:
Yes, I dislike it too, but make it this way just to try move to Retrofit. I need to use IOExcetion due to limitation of Retrofit, so I just do it this way this time as Proof of concept. I agree that for production code it should be the same contract.